### ISSUE TITLE
Redo sample app: order number is getting serialized weirdly

### ISSUE THREAD

**Issue by aboodman** [rocicorp/replicache-sdk-js#148]
```
22:58:20.698390 repm.go:64: <-- 200 http://localhost:7001/pull {"stateID":"ia2puj0gebgdvj6u367r3a1q52gvu361","lastMutationID":0,"patch":[{"op":"remove","path":"/"},{"op":"add","path":"/~1list~129395","value":{"id":29395,"ownerUserID":3}},{"op":"add","path":"/~1todo~11717351655","value":{"complete":false,"id":1717351655,"listId":29395,"order":89884656743115790000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000,"text":"todo one"}}],"checksum":"35478836","clientViewInfo":{"httpStatusCode":200,"errorMessage":""}}
```

**Comment by aboodman** [comment]
This issue exists in both lit-todo demo too.

**Comment by arv** [comment]
Let's just fix this and create an npm repo

**Comment by arv** [comment]
Fixed with 8da53e8c655dddc7dbaf847a286c684c41f39289
